modules/manage/pages/console/data-transforms.adoc

```diff
@@ -0,0 +1,49 @@
+= Manage Data Transforms in Redpanda Console
+:description: You can use Redpanda Console to monitor the status and performance metrics of your transform functions. You can also view detailed logs and delete transform functions when they are no longer needed.
+
+{description}
+
+== Prerequisites
+
+Before you begin, ensure that you have the following:
+
+- Redpanda Console installed.
+- xref:develop:data-transforms/configure.adoc#enable-transforms[Data transforms enabled] in your Redpanda cluster.
+
+[[monitor]]
+== Monitor transform functions
+
+To monitor transform functions in Redpanda Console:
+
+. Navigate to the *Transforms* menu.
+. Click on the name of a transform function to view detailed information:
+  - The partitions that the function is running on
+  - The broker (node) ID
+  - Any lag (the amount of pending records on the input topic that have yet to be processed by the transform)
+
+[[logs]]
+== View logs
+
+To view logs for a transform function in Redpanda Console:
+
+. Navigate to the *Transforms* menu.
+. Click on the name of a transform function
+. Click the *Logs* tab to see the logs.
```

Comment:
```suggestion
. Navigate to the *Transforms* menu.
. Click on the name of a transform function.
. Click the *Logs* tab to see the logs.
```

Reply:
Within these sections, sometimes there is end punctuation, sometimes not. Whichever you decide to go with, be consistent.